yt_dlp/extractor/youtube/_video.py

```diff
@@ -3648,6 +3648,12 @@ def feed_entry(name):
                     reason = f'{remove_end(reason.strip(), ".")}. {self._youtube_login_hint}'
                 elif get_first(playability_statuses, ('errorScreen', 'playerCaptchaViewModel', {dict})):
                     reason += '. YouTube is requiring a captcha challenge before playback'
+                elif "This content isn't available, try again later" in reason:
+                    reason = (
+                        f'{remove_end(reason.strip(), ".")}. {"Your account" if self.is_authenticated else "The current session"} has been rate-limited by YouTube for up to an hour. '
+                        'It is recommended to use `-t sleep` to add a delay between video requests to avoid hitting the rate limit. '
+                        'For more information, refer to  https://github.com/yt-dlp/yt-dlp/wiki/Extractors#this-content-isnt-available-try-again-later .'
+                    )
```

Comment:
since the rate limit is always by session, can we drop the conditional? also lines way too long
```suggestion
                    reason = (
                        f'{remove_end(reason.strip(), ".")}. Your current session has been rate-limited '
                        f'by YouTube for up to an hour. It is recommended to use `-t sleep` to add a delay '
                        f'between video requests to avoid hitting the rate limit. For more information, refer to  '
                        f'https://github.com/yt-dlp/yt-dlp/wiki/Extractors#this-content-isnt-available-try-again-later'
                    )
```